src/test/ui/associated-const/associated-const-no-item.stderr

```diff
@@ -1,12 +1,13 @@
 error[E0599]: no associated item named `ID` found for type `i32` in the current scope
   --> $DIR/associated-const-no-item.rs:5:23
    |
+LL | trait Foo {
+   | --------- `Foo` defines an item `ID`
+...
 LL | const X: i32 = <i32>::ID;
    |                       ^^ associated item not found in `i32`
    |
    = help: items from traits can only be used if the trait is implemented and in scope
-   = note: the following trait defines an item `ID`, perhaps you need to implement it:
-           candidate #1: `Foo`
```

Comment:
TODO: we should determine whether the implementing trait is in scope. If it isn't, suggest importing it. If it is, check whether the found adt implements it directly but the bounds weren't fulfilled. If not, suggest implementing it.